StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSession.swift

```diff
@@ -77,6 +99,8 @@ public extension StripeAPI {
         public let livemode: Bool
         @_spi(STP) public let paymentAccount: PaymentAccount?
         @_spi(STP) public let bankAccountToken: BankAccountToken?
+        let status: Status?
```

Comment:
looks like status is non-optional @kgaidis-stripe

Reply:
@vardges-stripe I believe if a merchant is NOT exposed into the flag, it actually returns a NULL (I know, confusing)